ts-nestjs-sequelize/.gitignore

```diff
@@ -0,0 +1,35 @@
+# compiled output
+/dist
+/node_modules
```

Comment:
These two should already be handled by the repo's root gitignore.

Reply:
I'll remove the .gitignore.